### PR TITLE
fix(ui): include all scripts in preselected commissioning except noauto

### PR DESCRIPTION
**Pull request by squidsoup**
## Done
The default commissioning script selector erroneously excluded non-default scripts (i.e. scripts provided by users), when it should include any script that is not marked `noauto`.

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Open the commissioning panel, both default scripts (those provided by MAAS), and custom user scripts should be selected by default. No scripts with the `noauto` label should be selected.

## Fixes

Fixes: #1893 

## Launchpad issue
[
lp#1904253](https://bugs.launchpad.net/maas/+bug/1904253)

## Backports

In general, please propose fixes against *master* rather than release branches (e.g. 2.7), unless the fix is only applicable for that specific release. Please apply backport labels to the PR (e.g. "Backport 2.7") for the appropriate releases to target.

Only bug and security fixes should be backported, new features should only land in master.

### CODE REVIEW

File: ui/src/app/store/scripts/selectors.ts [hunk 1]
```diff
@@ -89,16 +89,16 @@ const testingWithUrl = createSelector([testing], (testScripts) =>
 /**
  * Get scripts that match a term.
  * @param {RootState} state - The redux state.
- * @param {String} term - The term to match against.
- * @param {String} type - The type of script.
- * @returns {Scripts[]} A filtered list of scripts.
+ * @param term - The term to match against.
```
Comment (squidsoup) [inline ui/src/app/store/scripts/selectors.ts:92]: Just a drive-by here to remove some unneeded types from the jsdoc.

**Review by Caleb-Ellis — APPROVED** [review]
LGTM! I wonder if there's still some value in having the `defaultCommissioning` selector, though I guess it's just background noise if we don't use it anywhere.
